examples/semantic-cache/cache.py

```diff
@@ -0,0 +1,142 @@
+import os
+from datetime import datetime
+from typing import Optional, Annotated
+
+import requests
+import dotenv
+from fastapi import Depends, FastAPI
+from fastapi.security import HTTPAuthorizationCredentials, HTTPBearer
+from sqlmodel import (
+    SQLModel,
+    Session,
```

Comment:
Please use `AsyncSession`

Reply:
I removed all `async def` controller, so there is no need to use AsyncSession/AsyncEngine anymore.